pkg/test/ginkgo/external.go

```diff
@@ -123,9 +123,13 @@ func extractReleaseImageStream(logger *log.Logger, registryAuthFilePath string)
 		logger.Printf("Using target cluster release image %q", releaseImage)
 	}
 
+	extractStartTime := time.Now()
+	logger.Printf("Run image extract for release image %q at %v", releaseImage, extractStartTime)
 	if err := runImageExtract(releaseImage, "/release-manifests/image-references", tmpDir, registryAuthFilePath); err != nil {
 		return nil, fmt.Errorf("failed extracting image-references from %q: %w", releaseImage, err)
 	}
+	extractDuration := time.Since(extractStartTime)
+	logger.Printf("Run image extract completed for release image %q at %v", releaseImage, extractDuration)
```

Comment:
I would put this in runImageExtract itself instead of surrounding the call to it

Reply:
I'll let the current run finish and update the PR